implementations/code_df/issue_response_time_github.py

```diff
@@ -0,0 +1,114 @@
+from datetime import datetime
+import numpy as np
+
+import utils
+from issue import Issue
+
+
+class IssueResponseTimeGithub(Issue):
+    """
+    Class for Issue Response Time
+    """
+
+    def _flatten(self, item):
+        """
+        Flatten a raw issue fetched by Perceval into a flat dictionary.
+
+        A list with a single flat directory will be returned.
+        That dictionary will have the elements we need for computing metrics.
+        The list may be empty, if for some reason the issue should not
+        be considered.
+
+        :param item: raw item fetched by Perceval (dictionary)
+        :returns:   list of a single flat dictionary
+        """
+
+        creation_date = utils.str_to_date(item['data']['created_at'])
+        if self.since and (self.since > creation_date):
+            return []
+
+        if self.until and (self.until < creation_date):
+            return []
+
+        # all pull requests are considered to be issues by default.
+        # this check removes pull requests from the
+        # analysis.
+        if 'pull_request' in item['data']:
+            return []
+
+        flat = {
+            'repo': item['origin'],
+            'hash': item['data']['id'],
+            'category': "issue",
+            'author': item['data']['user']['login'],
+            'created_date': creation_date,
+            'current_status': item['data']['state']
+        }
+
+        member_comments = [comment for comment in item['data']['comments_data']
+                           if comment['author_association'] == 'MEMBER']
+
+        flat['issue_resp_time'] = np.NaN if len(member_comments) == 0 \
+            else (datetime.now()
+                  - utils.str_to_date(
```

Comment:
To find the first comment by a maintainer, I used the following logic. Please have a look and let me know if this is easy to understand / requires other changes. What I am doing here is this:
- Pickout the comments by maintainers and store in `member_comments`.
- Add the issue response time to `flat` by calculated the number of days between today and the first comment by a maintainer (using the `min`).

Reply:
I don't understand this rationale. Time should be from the moment the issue is opened, to the moment it receives the first reponse (not from the first comment to today). If there was still no comment, we should ignore the issue, since it is not correct to say it was responded today...

Are we in sync?